compiler/rustc_middle/src/mir/interpret/allocation.rs

```diff
@@ -673,15 +679,17 @@ impl InitMask {
             for block in (blocka + 1)..blockb {
                 self.blocks[block] = u64::MAX;
             }
-        } else {
+        } else if let Some(blocka_val) = self.blocks.get_mut(blocka) {
```

Comment:
Same, please add a comment saying why it is okay to do nothing here if `get_mut` returns `None`.